core/src/main/java/nu/yona/server/device/service/DeviceService.java

```diff
@@ -412,4 +414,32 @@ private Version parseVersionString(String versionStr)
 			throw DeviceServiceException.invalidVersionString(versionStr);
 		}
 	}
+
+	public void registerVpnStatusChangeEvent(UUID userId, UUID deviceId, boolean isVpnConnected)
```

Comment:
I first read this as "register to the event". Maybe `recordVpnStatusChangeEvent` is a name that can be less easy misread.

Reply:
I've tried reading it in an alternative way, but couldn't. I see "Register VPN status change event" and that's what it does. Also, I think that changing the verb from "register" into "record" won't help anybody.

Am I missing something?